.taskcluster.yml

```diff
@@ -1,6 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 version: 0
 allowPullRequests: collaborators
 tasks:
+####################################################################################################
+# Task: Pull requests
+####################################################################################################
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        env: true
```

Comment:
This is probably not needed

Reply:
Probably not, but it's convenient to have these environment variables ready if and when they're needed.